fortitude/src/check.rs

```diff
@@ -228,7 +229,7 @@ fn parse_config_file(config_file: &Option<PathBuf>) -> Result<CheckSettings> {
 }
 
 /// Get the list of active rules for this session.
-fn ruleset(args: RuleSelection, preview: &PreviewMode) -> anyhow::Result<Vec<Rule>> {
+fn ruleset(args: RuleSelection, preview: &PreviewMode) -> anyhow::Result<RuleTable> {
```

Comment:
To avoid confusion with `RuleSet`, I think this function should be renamed `rule_table`.